iOS/MMIWG2S/ViewControllers/IntroViewController.swift

```diff
@@ -0,0 +1,219 @@
+            imageView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+        
+        
+        let gradient = self.gradient
+        let gradientView = UIImageView(image:gradient)
+        gradientView.frame = view.frame
+        gradientView.contentMode = .scaleAspectFill
+        gradientView.clipsToBounds = true
+        view.addSubview(gradientView)
+        gradientView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            gradientView.topAnchor.constraint(equalTo: view.topAnchor),
+            gradientView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            gradientView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            gradientView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+
+        setupUI()
+    }
+    
+    func setupUI() {
+        
+        //Setup UI
+        let labelMMIW = UILabel(frame:CGRect(x:0,y:0,width:50,height:21))
+        let labelMMIWInfo = UILabel(frame:CGRect(x:0,y:0,width:200,height:200))
+        let buttonNext = UIButton(frame:CGRect(x:0,y:0, width:30, height:17))
+ 
+        let spacerView1 = UIView()
+        let spacerView2 = UIView()
+
+        labelMMIW.translatesAutoresizingMaskIntoConstraints = false
+        spacerView1.translatesAutoresizingMaskIntoConstraints = false
+
+        labelMMIWInfo.translatesAutoresizingMaskIntoConstraints = false
+        spacerView2.translatesAutoresizingMaskIntoConstraints = false
+
+        buttonNext.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(labelMMIW)
+        view.addSubview(labelMMIWInfo)
+        view.addSubview(spacerView1)
+
+        pc1View.translatesAutoresizingMaskIntoConstraints = false
+        pc2View.translatesAutoresizingMaskIntoConstraints = false
+        pc3View.translatesAutoresizingMaskIntoConstraints = false
+        pc4View.translatesAutoresizingMaskIntoConstraints = false
+    
+        labelMMIW.textAlignment = .center
+        labelMMIW.text = NSLocalizedString("slideviewheader", comment: "Text")
+        labelMMIW.textColor = UIColor.white
+        labelMMIW.font = UIFont(name: "Roboto-Regular", size:36.0 )
+
+        labelMMIWInfo.numberOfLines = 0
+        labelMMIWInfo.textColor = UIColor.white
+        labelMMIWInfo.font = UIFont(name: "Roboto-Regular", size:25.0 )
+        labelMMIWInfo.text = NSLocalizedString(mmiwtext, comment: "text")
+        //initialize pcs
+        pc1View.backgroundColor = UIColor.white
+        pc2View.backgroundColor = UIColor.white
+        pc3View.backgroundColor = UIColor.white
+        pc4View.backgroundColor = UIColor.white
+
+        switch self.page {
+        case 0:
+            pc1View.backgroundColor = UIColor.red
+        case 1: pc2View.backgroundColor = UIColor.red
+        case 2: pc3View.backgroundColor = UIColor.red
+        case 3: pc4View.backgroundColor = UIColor.red
+        default: pc1View.backgroundColor = UIColor.red
```

Comment:
were these in an array, we could simply say `pageControlViews[page] = .red` and remove the switch.

Reply:
I did this: pageControlViews[page].backgroundColor = .red